doc/api/next_api_changes/removals/23XXX-GL.rst

```diff
@@ -0,0 +1,4 @@
+Get/set window title methods have been removed from the canvas
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the corresponding methods on the FigureManager instead.
```

Comment:
Maybe mention also that one can use the native methods of a GUI framework if cross-backend compatibility is not needed.